`azurerm_app_configuration_key`, `azurerm_app_configuration_feature` - change resource ID to Data Plane URL format (#20082)
hashicorp/terraform-provider-azurerm: website/docs/r/app_configuration_key.html.markdown

App Configuration Keys can be imported using the `resource id`, e.g.

```shell
terraform import azurerm_app_configuration_key.test https://appconfname1.azconfig.io/kv/keyName?label=labelName
```

> If you wish to import a key with an empty label then substitute the label's name with empty, like this:

Reviewer: grammer
```suggestion
If you wish to import a key with an empty label then simply leave label's name blank:
```
